### DISCUSSION TITLE
SD upscalers VS Gigapixel AI

### DISCUSSION THREAD

**Discussion by thedyze** [hlky/stable-diffusion-webui#1022] [General]
Just tried the Gigapixel trial (https://www.topazlabs.com/gigapixel-ai), must say I'm super impressed with the result, escpecially the speed of the processing.

Curious how it can be so fast, generating a 8600x11500 upscale in 2-3 seconds with a result better than e.g RealESRGAN, locally.
Wondering how there can be such large differences in processing time.

**Comment by e850205** [comment]
Money. Money is how.
